docs/Using Fleet/Vulnerability-Processing.md

```diff
@@ -1,14 +1,8 @@
 # Vulnerability processing
 
-- [What to expect](#what-to-expect)
-- [Coverage](#coverage)
-- [Configuration](#configuration)
-- [Performance](#performance)
-- [Detection pipeline](#detection-pipeline)
+## Overview
```

Comment:
```suggestion
## Introduction
```

Should we start being consistent? I see we use "Introduction" here on Enroll hosts: https://fleetdm.com/docs/using-fleet/enroll-hosts